bundles/org.openhab.binding.dbquery/src/main/java/org/openhab/binding/dbquery/action/DBQueryActions.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dbquery.action;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.dbquery.internal.DatabaseBridgeHandler;
+import org.openhab.binding.dbquery.internal.QueryHandler;
+import org.openhab.binding.dbquery.internal.domain.ExecuteNonConfiguredQuery;
+import org.openhab.binding.dbquery.internal.domain.QueryResult;
+import org.openhab.binding.dbquery.internal.domain.ResultRow;
+import org.openhab.binding.dbquery.internal.error.UnnexpectedCondition;
+import org.openhab.core.automation.annotation.ActionInput;
+import org.openhab.core.automation.annotation.RuleAction;
+import org.openhab.core.thing.binding.ThingActions;
+import org.openhab.core.thing.binding.ThingActionsScope;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Joan Pujol - Initial contribution
+ */
+@ThingActionsScope(name = "dbquery")
+@NonNullByDefault
+public class DBQueryActions implements IDBQueryActions, ThingActions {
+    private final Logger logger = LoggerFactory.getLogger(DBQueryActions.class);
+
+    private @Nullable QueryHandler queryHandler;
+    private @Nullable DatabaseBridgeHandler databaseBridgeHandler;
+
+    @Override
+    public void activate() {
+        logger.trace("activate actions");
+    }
+
+    @Override
+    public void deactivate() {
+        logger.trace("deactivate actions");
+    }
```

Comment:
What is the point of these? They dont seem to serve a purpose as there is no code... See guidelines here on not using the logs instead of the debugger.

https://www.openhab.org/docs/developer/guidelines.html#f-logging